packages/frontend/src/components/UserSettings/UsersAndGroupsPanel/UsersView.tsx

```diff
@@ -398,7 +398,7 @@ const UsersView: FC = () => {
                                     disabled={
                                         user.data?.userUuid ===
                                             orgUser.userUuid ||
-                                        organizationUsers.length <= 1
+                                        organizationUsers.length < 1
```

Comment:
Perhaps we can pass a new param to `useOrganizationUsers` to then set the total number of users on `select`, regardless of the search input. So we can

```ts
  disabled={
                                        user.data?.userUuid ===
                                            orgUser.userUuid || totalNumberOfUsers === 1
```

Reply:
@joaoviana and @ZeRego, the admin user can never delete themself from the UI, so there will always be at least one. Isn't that sufficient? If you would like to be more careful here, we can, but I think this case is handled. 

You also can't change the last admin to not be an admin anymore

<img width="875" alt="Screenshot 2024-03-04 at 15 22 01" src="https://github.com/lightdash/lightdash/assets/1864179/9c862001-6b51-4e63-a02d-f058801cbe7e">

Reply:
Oh I see @magnew

I just tested this: 
- Couldnt' delete myself
- Searched for me and couldn't delete myself
- Searched for others and could delete them
- Deleted everyone except me 
- Couldn't delete myself, and couldn't delete myself while searching